upgrade.php

```diff
@@ -1623,7 +1622,53 @@ function upgrade_1824_sqlite() {
           {PRIMARY} (`on_vacation`,`notified`),
           CONSTRAINT `vacation_notification_pkey` FOREIGN KEY (`on_vacation`) REFERENCES `vacation` (`email`) ON DELETE CASCADE);
     ");
+}
+
+function upgrade_1825_mysql_pgsql() {
+    # Add support for multiple servers
+    $server = table_by_key('server');
+    db_query_parsed("
+        CREATE TABLE {IF_NOT_EXISTS} $server (
+          server VARCHAR(100) {LATIN1} NOT NULL,
+          description VARCHAR(255) {LATIN1} NOT NULL,
+          address VARCHAR(100) {LATIN1} NOT NULL,
+          created {DATE},
+          modified {DATECURRENT},
+          PRIMARY KEY (server)
+        ) {MYISAM} ;
+    ");
+}
+
+function upgrade_1825_sqlite() {
+    $server = table_by_key('server');
+    db_query_parsed("
+        CREATE TABLE $server (
+         `server` varchar(100) NOT NULL,
+         `description` varchar(255) NOT NULL,
+         `address` varchar(100) NOT NULL,
+         `created` {DATE},
+         `modified` {DATECURRENT},
+         {PRIMARY} (`email`));
```

Comment:
I'd prefer if you can create one function for all database types. (Add additional {...} markers if needed.)

This will also avoid problems like the {PRIMARY} (`email`) which refers to a non-existing field ;-)

Also, description should probably be {UTF8}.